pages/common/qcp.md

```diff
@@ -0,0 +1,16 @@
+# qcp
+
+> Copy files with your favorite text editor.
+> More information: <https://www.nongnu.org/renameutils/>.
+
+- Copy a single file (invoke an editor with source on the left, target on the right):
+
+`qcp {{source_file}}`
+
+- Copy multiple JPG files:
+
+`qcp {{*.jpg}}`
+
+- Copy files, but swap pattern in the editor -- target on the left and source on the right:
```

Comment:
@waldyrious @sbrl - Is "--" the right way to express a pause in a sentence ?

Reply:
Or should I rephrase the whole sentence, like this:

    - Copy files, but swap position of source and target in the editor:

Reply:
Sort of, @agnivade. Personally I'd use a single dash ` - ` there, not a double one - or, of course, ....`editor, with the target`..... instead.

Reply:
@sbrl thanks for correcting them.